pkg/ddevapp/ddevapp_test.go

```diff
@@ -3517,13 +3517,19 @@ func TestMultipleComposeFiles(t *testing.T) {
 	files, err := app.ComposeFiles()
 	assert.NoError(err)
 	require.NotEmpty(t, files)
-	assert.Equal(5, len(files))
+	assert.Equal(6, len(files))
 	require.Equal(t, app.GetConfigPath(".ddev-docker-compose-base.yaml"), files[0])
 	require.Equal(t, app.GetConfigPath("docker-compose.override.yaml"), files[len(files)-1])
 
 	require.NotEmpty(t, app.ComposeYaml)
 	require.True(t, len(app.ComposeYaml) > 0)
 
+	dockerIP, err := dockerutil.GetDockerIP()
+	require.NoError(t, err)
+	if app.BindAllInterfaces {
+		dockerIP = "0.0.0.0"
+	}
```

Comment:
Is the test covering the default situation, with regular 127.0.0.1?

Reply:
Yes, you are right. I should check for `127.0.0.1` in the test, not use `dockerutil.GetDockerIP()` here.